### ISSUE TITLE
The Argent Tournament quest chain bugged/cutted

### ISSUE THREAD

**Issue by HelljudgeWhitemane** [gamefreedomgit/Maelstrom#2639]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
The Argent Tournament quest chain:

I - The Argent Tournament
Welcome to the Argent Tournament - a gathering of heroes from around Azeroth on the Lich King's doorstep in Icecrown. As the Argent Crusade continues their efforts against the Scourge, they also promote competition through the system of mounted combat. Grab a lance, hop on a mount, and hold on tight.

1) [The Argent Tournament](https://www.wowhead.com/wotlk/quest=13668)
2) The Mastery Quests
--- [Mastery of Melee](https://www.wowhead.com/wotlk/quest=13829)
--- [Mastery of the Shield-Breaker](https://www.wowhead.com/wotlk/quest=13838)
--- [Mastery of the Charge](https://www.wowhead.com/wotlk/quest=13839)

II - Aspirant
Now that you have learned the basics of mounted combat, you must prove yourself further in both mounted combat practice, unmounted Scourge combat, and adventuring to find ancient artifacts from maidens in lakes before you have the right to represent your faction in the Tournament.

3) [Up To The Challenge](https://www.wowhead.com/wotlk/quest=13678)
You must complete the following daily quests to gain Aspirant's Seals:
--- i - [Training in the Field](https://www.wowhead.com/wotlk/quest=13676)
--- ii - [Learning the Reins](https://www.wowhead.com/wotlk/quest=13677)
--- iii - One of the following daily blade quests (different every day):
------ [A Blade Fit For A Champion](https://www.wowhead.com/wotlk/quest=13673)
------ [A Worthy Weapon](https://www.wowhead.com/wotlk/quest=13674)
------ [The Edge of Winter](https://www.wowhead.com/wotlk/quest=13675)

I've completed all quests that leading me to 3) [Up To The Challenge](https://www.wowhead.com/wotlk/quest=13678) , but then there no following daily quests from Sunreaver NPCs to gain Aspirant's Seals, even after i accept quest [Up To The Challenge](https://www.wowhead.com/wotlk/quest=13678) , nothing happens
2 screenshots https://imgur.com/a/EtLm3hk

**How it should work:**
After completing The Mastery Quests chain, following daily quests to gain Aspirant's Seals from Sunreaver NPCS for the quest [Up To The Challenge](https://www.wowhead.com/wotlk/quest=13678) should be avaliable

**Comment by emirie1** [comment]
I have encountered this same issue. At first I thought that it might be an issue with me being a goblin which was not an available race during the tournament and I believe had to be grandfathered in to start quests there similar to worgen. However, I purchased a race change and swapped to orc which should have allowed me to become an Aspirant of Orgrimmar but the only available quest is "Up To The Challenge" as mentioned above.

This is okay for anyone who doesn't care about collecting things like mounts/pets/tabards/titles/exalted reputations for their associated achievements and rewards, but for someone like myself who is a collector of these things this leaves the only available way to earn Champion's Seals being from the last boss of Trial of the Champion Normal, all bosses in Trial of the Champion Heroic (once per day), and I believe bosses in Trial of the Crusade 10/25/10HC/25HC. It also completely prevents the ability to earn reputation with The Sunreavers/The Silver Covenant.
